foreshadow/utils/validation.py

```diff
@@ -102,16 +148,16 @@ def check_transformer_imports(printout=True):
             transformers
 
     """
-    import foreshadow.transformers.concrete as conc
```

Comment:
We didn't really have unit tests around this, does this still work?

Reply:
No idea.